cluster/gce/config-common.sh

```diff
@@ -130,22 +130,22 @@ NODE_SCOPES="${NODE_SCOPES:-monitoring,logging-write,storage-ro}"
 # Root directory for Kubernetes files on Windows nodes.
 WINDOWS_K8S_DIR="C:\etc\kubernetes"
 # Directory where Kubernetes binaries will be installed on Windows nodes.
-WINDOWS_NODE_DIR="${WINDOWS_K8S_DIR}\node\bin"
+export WINDOWS_NODE_DIR="${WINDOWS_K8S_DIR}\node\bin"
 # Directory where Kubernetes log files will be stored on Windows nodes.
-WINDOWS_LOGS_DIR="${WINDOWS_K8S_DIR}\logs"
+export WINDOWS_LOGS_DIR="${WINDOWS_K8S_DIR}\logs"
 # Directory where CNI binaries will be stored on Windows nodes.
-WINDOWS_CNI_DIR="${WINDOWS_K8S_DIR}\cni"
+export WINDOWS_CNI_DIR="${WINDOWS_K8S_DIR}\cni"
 # Directory where CNI config files will be stored on Windows nodes.
-WINDOWS_CNI_CONFIG_DIR="${WINDOWS_K8S_DIR}\cni\config"
+export WINDOWS_CNI_CONFIG_DIR="${WINDOWS_K8S_DIR}\cni\config"
 # Pod manifests directory for Windows nodes on Windows nodes.
-WINDOWS_MANIFESTS_DIR="${WINDOWS_K8S_DIR}\manifests"
+export WINDOWS_MANIFESTS_DIR="${WINDOWS_K8S_DIR}\manifests"
 # Directory where cert/key files will be stores on Windows nodes.
-WINDOWS_PKI_DIR="${WINDOWS_K8S_DIR}\pki"
+export WINDOWS_PKI_DIR="${WINDOWS_K8S_DIR}\pki"
 # Path for kubelet config file on Windows nodes.
-WINDOWS_KUBELET_CONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet-config.yaml"
+export WINDOWS_KUBELET_CONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet-config.yaml"
 # Path for kubeconfig file on Windows nodes.
-WINDOWS_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.kubeconfig"
+export WINDOWS_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.kubeconfig"
 # Path for bootstrap kubeconfig file on Windows nodes.
-WINDOWS_BOOTSTRAP_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.bootstrap-kubeconfig"
+export WINDOWS_BOOTSTRAP_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.bootstrap-kubeconfig"
 # Path for kube-proxy kubeconfig file on Windows nodes.
-WINDOWS_KUBEPROXY_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubeproxy.kubeconfig"
+export WINDOWS_KUBEPROXY_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubeproxy.kubeconfig"
```

Comment:
are these read by anything? why do we need to export them?

Reply:
cc @yujuhong

Reply:
These are used in https://github.com/kubernetes/kubernetes/blob/3a501844219d06b6fd777a911175b823ab146d43/cluster/gce/util.sh#L1525.

Reply:
Yes, thanks for your help. @pjh

Reply:
please add a comment somewhere in this file about this, so in the future if they are not used we can remove them

Reply:
gentle bump

Reply:
Got it, sorry for late reply. Please check again.